CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [1.1.0] - 2023-05-15 - Giuseppe Lo Presti <lopresti@cern.ch>
+
+* Added a new `/invite-accepted` endpoint to support an invitation
+  workflow in the context of the ScienceMesh.
+* Officially added the `/ocm-provider` discovery endpoint, already
+  in use by several implementations. Within this endpoint, clarified
+  which are the minimal capabilities required to be "OCM compliant".
+* Added support for multi-protocol shares, and fully specified
+  the properties required for each supported protocol.
+* Added a `federation` recipient share type.
+* Deprecated `protocol.options` in `/shares`.
```

Comment:
Should this be included in the release notes? This is not part of the diff from v1.0 to v1.1, right?

Reply:
Well, there are no release notes in this repo, nor was there a changelog - I created it to keep track of the versions going forward!

Reply:
...Now I see what you meant, I've attached this text to the GitHub release.